inspectors/services/elementrepositoryservice.cpp

```diff
@@ -40,6 +42,7 @@ QList<Ms::Element*> ElementRepositoryService::findElementsByType(const Ms::Eleme
     case Ms::ElementType::STAFF: return findStaffs();
     case Ms::ElementType::LAYOUT_BREAK: return findSectionBreaks(); //Page breaks and line breaks are of type LAYOUT_BREAK, but they don't appear in the inspector for now.
     case Ms::ElementType::PEDAL: return findPedals();
+    case Ms::ElementType::CLEF: return findCourtesyClefs(); //"Show Courtesy clef" property can be applied only to courtesy clefs
```

Comment:
I would recommend transferring this comment to the model, where this property is used

and here I would write something like this - "Within the inspector, we use only this type of clefs, see the details in such a model"